cc.cubocore.CoreTime.yml

```diff
@@ -0,0 +1,44 @@
+app-id: cc.cubocore.CoreTime
+runtime: org.kde.Platform
+runtime-version: '6.8'
+sdk: org.kde.Sdk
+command: coretime
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --share=network # Required to show weather info
+  - --socket=wayland
+  - --filesystem=home
+  - --filesystem=host
```

Comment:
these two filesystem permission are very broad and redundant (one includes the other)